### ISSUE TITLE
Acknowledgments broken by missing new line

### ISSUE THREAD

**Issue by jochym** [dressedfez/myst_template_physical_review_journals#18]
The acknowledgments are broken by a missing newline after comment ending glossary printing. I have fixed it by adding a newline before \begin{acknowledgments} in the template, but this is rather a workaround, not a fix. It should be fixed in mystmd converter itself. But I do not know how. Maybe you can communicate to the mystmd team about this issue (I have submitted an issue to mystmd as well   https://github.com/executablebooks/mystmd/issues/737)

**Comment by rowanc1** [comment]
Thank you for the issue!

**Comment by jochym** [comment]
Apparently mystmd guys are aiming for fixing ASAP. Just for people searching for the solution to this problem:

You can fix it temporarily by adding new line before `\begin{acknowledgments}` in the  `_build/templates/tex/dressedfez/physical_review_journals/template.tex` file in your project directory.

Here is the fixed fragment:
```
[-CONTENT-]

[#- if parts.acknowledgments #]

\begin{acknowledgments}
[-parts.acknowledgments-]
\end{acknowledgments}
[# endif -#]

```

**Comment by dressedfez** [comment]
I will add the fix in the template unless @rowanc1 has a fix at hand. 

Update: it seems there is already at fix on its way. I hence will not touch the template. Happy to see people are using it.

**Comment by rowanc1** [comment]
A fix was released in v1.1.27! Thanks both for your help reporting these and also super stoked to see these templates getting used. 🚀

**Comment by dressedfez** [comment]
@jochym  can you please let us know if this resolved your issue.
